packages/labs/gen-wrapper-react/src/index.ts

```diff
@@ -10,7 +10,9 @@ import {
   PackageJson,
   LitElementDeclaration,
   ModuleWithLitElementDeclarations,
+  getImportsStringForReferences,
 } from '@lit-labs/analyzer';
+import {Event as ModelEvent} from '@lit-labs/analyzer/lib/model.js';
```

Comment:
Naming (here and in Vue): `EventModel` reads better

Reply:
Done.